.circleci/run_release_workflow.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [[ -z "${CIRCLE_TOKEN}" ]]; then
+  echo Cannot trigger release workflow. CircleCI user token not found.
+  exit 1
+fi
+
+BRANCH=1.3.x
+
```

Comment:
Why isn't the branch check in here like with CashProof and MarketFacingData?

Reply:
Okay, makes sense, why didn't we do what's done here on MF and Proof?

Reply:
Sensible ideas don't all come to me at once 🤷‍♂️